### ISSUE TITLE
Listbox with grouped options reported as error in AI

### ISSUE THREAD

**Issue by Anschur** [microsoft/accessibility-insights-web#4142]
## Describe the bug

When options in a listbox are grouped in elements using role="group" Accessibility Insights will complain that the listbox is missing required children option, and that that the options are missing required parent listbox.

**To Reproduce**

1. Go to https://w3c.github.io/aria-practices/examples/listbox/listbox-grouped.html
2. Run AI
3. Observe errors on listbox structure

## CodePen repro example
https://w3c.github.io/aria-practices/examples/listbox/listbox-grouped.html

## Expected behavior

I expected no errors on this w3c example.

## Context (please complete the following information)

- OS Name & Version: Windows 10.0.19042 Build 19042
- AI-Web Version & Environment: 2.25.0
- Browser Version: Edge 90.0.818.42 (Official build) (64-bit)
- Target Page: https://w3c.github.io/aria-practices/examples/listbox/listbox-grouped.html

## Are you willing to submit a PR?

No

## Did you search for similar existing issues?

Yes

**Comment by peterdur** [comment]
Looks like https://github.com/dequelabs/axe-core/issues/2523 might be related.

**Comment by ferBonnin** [comment]
Agree with @peterdur this is the same issue as [#2523](https://github.com/dequelabs/axe-core/issues/2523). ARIA 1.2 is not an official recommendation yet.

**Comment by dbjorge** [comment]
Fixed some time ago in axe-core by https://github.com/dequelabs/axe-core/pull/3195
